pages/wiki/technical.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Technical Details
+layout: documentation
+---
+This page contains technical details about currently supported watches.
+
+# AsteroidOS Watch Technical Details
+---
+
+While AsteroidOS does not use Android, it does rely on drivers that are be supported with [libhybris](https://github.com/libhybris/libhybris).
```

Comment:
"that are be supported" to
"that are supported when using" (Which would hint at mainline not using libhybis) or
"that are supported by"

Or a completely different approach, imo clearer:
"Although AsteroidOS does not use Android, it relies on libhybris to use the Android drivers."

Reply:
Good point.  I changed it to "While AsteroidOS does not use Android, it does rely on Android drivers that are, in the context of AsteroidOS, supported with [libhybris](https://github.com/libhybris/libhybris)."